src/main/java/graphql/GraphQL.java

```diff
@@ -266,19 +267,28 @@ public Builder executionIdProvider(ExecutionIdProvider executionIdProvider) {
         }
 
         /**
-         * For performance reasons you can opt into situation where the default instrumentations (such
-         * as {@link graphql.execution.instrumentation.dataloader.DataLoaderDispatcherInstrumentation} will not be
-         * automatically added into the graphql instance.
-         * <p>
-         * For most situations this is not needed unless you are really pushing the boundaries of performance
+         * As of GraphQL Java 22 there are no default instrumentations. Before that a DataLoaderDispatcherInstrumentation
+         * was added by default.
          * <p>
-         * By default a certain graphql instrumentations will be added to the mix to more easily enable certain functionality.  This
-         * allows you to stop this behavior
+         * For GraphQL Java 22 calling this method is equal to calling {@link #doNotAutomaticallyDispatchDataLoader()}
          *
          * @return this builder
          */
         public Builder doNotAddDefaultInstrumentations() {
```

Comment:
I would deprecate this method.  Its no longer useful and semantically weird without

Reply:
will remove it as discussed